client/components/button/style.scss

```diff
@@ -335,33 +335,4 @@ button {
 .dialog__content input[type='submit']::-moz-focus-inner {
 	border: 0;
 	padding: 0;
-}
-
-// ==========================================================================
-// Deprecated styles
-// ==========================================================================
-
-.button.is-link {
```

Comment:
I'm not sure why this is labelled as deprecated but it's used a few times throughout Calypso. For example, this branch:

![Screenshot_20190401-070712](https://user-images.githubusercontent.com/43215253/55306714-071bfb00-544d-11e9-9f28-bf299e786089.jpg)

Compared to:

![Screenshot_20190401-070807](https://user-images.githubusercontent.com/43215253/55306728-0f743600-544d-11e9-8125-03946df6c0cb.jpg)

Reply:
Thanks for catching this, @torres126! 

I dropped a similar question [earlier](https://github.com/Automattic/wp-calypso/pull/31909#pullrequestreview-220929672), but apparently GitHub seems to require self-reviews to be explicitly clicked "twice". :) 

Anyway, getting on to the discussion, that's a good catch! Maybe someone from @Automattic/design could advise on the usage of these styles.

Reply:
I have reversed removal the deprecated styles section as it causes a lot of unintended changes. Details on where these styles are used - https://github.com/Automattic/wp-calypso/pull/31909#issuecomment-479883061